packages/chain/cmt_log/cmt_log_rapid_test.go

```diff
@@ -74,26 +72,26 @@ func newCmtLogTestRapidSM(t *rapid.T) *cmtLogTestRapidSM {
 	sm.genNodeID = rapid.SampledFrom(gpaNodeIDs)
 	//
 	// Start it.
-	sm.l1Chain = append(sm.l1Chain, sm.nextAliasOutputWithID(0))
+	sm.l1Chain = append(sm.l1Chain, sm.nextStateAnchorWithStateIndex(0))
 	for _, nid := range gpaNodeIDs {
 		sm.l1Delivered[nid] = -1
 	}
 	return sm
 }
 
-func (sm *cmtLogTestRapidSM) nextAliasOutputWithID(stateIndex uint32) *isc.StateAnchor {
-	sm.genAOSerial++
-	var outputID iotago.ObjectID // TODO -> ObjectRef
-	binary.BigEndian.PutUint32(outputID[:], sm.genAOSerial)
-
-	anchor := iscmovetest.RandomAnchor(iscmovetest.RandomAnchorOption{StateMetadata: &[]byte{}, StateIndex: &stateIndex})
-	stateAnchor := isc.NewStateAnchor(
-		&iscmove.AnchorWithRef{
-			Object:    &anchor,
-			ObjectRef: sm.anchorRef,
-			Owner:     sm.stateAddress.AsIotaAddress(),
-		}, *cryptolib.NewRandomAddress().AsIotaAddress())
-
+// simulate StateAnchor to state transition
+func (sm *cmtLogTestRapidSM) nextStateAnchorWithStateIndex(stateIndex uint32) *isc.StateAnchor {
+	newObjID := sm.anchorRef.ObjectID.Clone()
+	newAnchor := iotago.ObjectRef{
+		ObjectID: &newObjID,
```

Comment:
```suggestion
		ObjectID: &sm.anchorRef.ObjectID,
```

(no need to clone it since it's immutable).